cfgov/unprocessed/js/apps/financial-well-being/fwb-results.js

```diff
@@ -1,72 +1,112 @@
 /* eslint no-extra-semi: "off" */
 'use strict';
 
-var Analytics = require( '../../modules/Analytics' );
-var Expandable = require( '../../organisms/Expandable' );
+const Analytics = require( '../../modules/Analytics' );
+const Expandable = require( '../../organisms/Expandable' );
 
-var expandableDom = document.querySelectorAll( '.content .o-expandable' );
-var expandable;
+const expandableDom = document.querySelectorAll( '.content .o-expandable' );
+let expandable;
 if ( expandableDom ) {
-  for ( var i = 0, len = expandableDom.length; i < len; i++ ) {
+  for ( let i = 0, len = expandableDom.length; i < len; i++ ) {
     expandable = new Expandable( expandableDom[i] );
     expandable.init();
   }
 }
 
+/**
+ * Initialize the results interactions
+ */
 function init() {
 
-  var toggleButtons = document.querySelectorAll(
+  const toggleButtons = document.querySelectorAll(
     '.comparison-chart_toggle-button'
   );
 
+  /**
+   * Changes the viewability of the results by category
+   * based on user input
+   * @param {string} category The category to display
+   */
   function switchComparisons( category ) {
-    var allCategories = document.querySelectorAll( '.comparison_data-point' );
-    var selectedButtonClass = 'comparison-chart_toggle-button__selected';
+    const allCategories = document.querySelectorAll( '.comparison_data-point' );
+    const showCategory = document.querySelectorAll(
+      '[class^="comparison_data-point ' + category + '"]'
+    );
+    const selectedButton = document.querySelector(
+      '[data-compare-by="' + category + '"]'
+    );
+    const selectedButtonClass = 'comparison-chart_toggle-button__selected';
+    const hiddenClass = 'u-hidden';
 
     // Hide all categories ...
     [].forEach.call( allCategories, function( el ) {
-      el.style.display = 'none';
+      el.classList.add( hiddenClass );
```

Comment:
Don't we need the `className` fallback for browsers that don't support `classList` (IE9)? (Here and 3 times below).

Reply:
Nope, we have a polyfill already in place.

Reply:
Ah, I see. Looking at the source, it seems like we should update that, since we manually pasted it in and it's now several years out of date. Perhaps we should install it via npm? (It seems poorly updated on the actual npm registry vs. what's in the repo, but we could reference the Git repo.)

Reply:
I can get onboard with that.